main/redis/redis.pre-install

```diff
@@ -2,5 +2,6 @@
 
 addgroup -S redis 2>/dev/null
 adduser -S -D -H -h /var/lib/redis -s /bin/false -G redis -g redis redis 2>/dev/null
+install -o redis -g redis -m 0770 -d /run/redis
```

Comment:
when package changes you need to increase pkgrel

Reply:
Good catch. Fixed

Reply:
@clandmeter this is needed for redis to work under docker. Dockerised services should be run in the foreground not as a daemon managed by initd. See [Remind issue 9113](https://bugs.alpinelinux.org/issues/9113) and linked issues for more info.

Reply:
See https://git.alpinelinux.org/cgit/aports/commit/?id=c4d4e9aa7abd163db3a9a4c4e6d6ba4f84d3a64f